__tests__/app.test.js

```diff
@@ -12,22 +12,47 @@ afterAll(() => {
     return db.end();
 });
 
-describe('/api/topics', () => {
-    describe('GET', () => {
-        it('returns an array of topics that contain a description and a slug property', () => {
-            return request(app)
-                .get('/api/topics')
-                .expect(200)
-                .then(({ body: { topics } }) => {
-                    expect(topics).toHaveLength(3);
-                    topics.forEach((topic) => {
-                        expect.objectContaining({
-                            description: expect.any(String),
-                            slug: expect.any(String)
-                        })
-                    });
-            })
-        });
+describe('GET', () => {
+    describe("/api/topics", () => {
+      it("returns an array of topics that contain a description and a slug property", () => {
+        return request(app)
+          .get("/api/topics")
+          .expect(200)
+          .then(({ body: { topics } }) => {
+            expect(topics).toHaveLength(3);
+            topics.forEach((topic) => {
+              expect(topic).toEqual(
+                expect.objectContaining({
+                  description: expect.any(String),
+                  slug: expect.any(String),
+                })
+              );
+            });
+          });
+      });
+    });
+    describe("/api/articles", () => {
+      it("returns an array of article objects", () => {
```

Comment:
This is all really good, I might add a test to make sure they are coming back in the right order, but this will probs pass right away as your SQL statement looks all good 👍